tesseract_core/sdk/cli.py

```diff
@@ -303,6 +303,12 @@ def build_image(
             help="Only generate the build context and do not actually build the image."
         ),
     ] = False,
+    gitignore: Annotated[
+        bool,
+        typer.Option(
+            help="Use .gitignore file to exclude files from the build context."
```

Comment:
This docstring can be scoped a little more accurately, right? "Respect .gitignore file when copying local dependencies to the build context"?

Reply:
That is, it currently seems to imply that it will use "this" repository's .gitignore to exclude files that might be contained under the tesseract directory.